dotnet-desktop-guide/net/winforms/printing/how-to-print-text-document.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "How to print multi-page text file"
+description: Learn how to print multiple page text file (Windows Forms .NET).
+ms.date: "04/29/2022"
+dev_langs: 
+  - "csharp"
+  - "vb"
+helpviewer_keywords: 
+  - "printing [Windows Forms .NET], printing multiple pages"
+  - "text [Windows Forms .NET], printing Windows Forms"
+  - "Windows Forms .NET, printing text"
+  - "printing [Windows Forms .NET], text"
```

Comment:
Lets keep these the same. I don't know how they affect the help viewer F1 key in visual studio.
```suggestion
  - "printing [Windows Forms], printing multiple pages"
  - "text [Windows Forms], printing Windows Forms"
  - "Windows Forms, printing text"
  - "printing [Windows Forms], text"
```

Reply:
ok.